darwin-deploy.sh

```diff
@@ -25,12 +25,12 @@ fi
 
 # Express should be installed
 
-if (which node)
+if (which express)
 then 
- echo "node is installed"
+ echo "express is installed"
 else
-  echo "installing node"
-brew install node
+  echo "installing express"
+brew install express
```

Comment:
Express is actually a javascript library. So you should follow this guide on how to install express as a javascript npm dependency in a javascript project:

https://expressjs.com/en/starter/installing.html

Reply:
I know this, I just forgot to change the pkg manager to npm. I even tested it with npm, and then copied the node if statement, and didn't change the most important part. Thank you sir